### PR TITLE
Style/format

### PR DESCRIPTION
**Pull request by Tristan-C789**
### Added a formatter

It is a best practice to follow a format guideline. To ensure this, we have added prettier, an automatic formatter, and use it to check if the format is good in the CI.

Before every PR, one should make sure to run `npx prettier --write ./app`!

### CODE REVIEW

File: app/screens/auth/forgot_password_screen.tsx [hunk 1]
```diff
@@ -1,121 +1,135 @@
...
+      <View style={styles.colContainer}>
+        {/* Input */}
+        <Text style={styles.text}>Email</Text>
+        <TextInput
+          style={styles.input}
+          placeholder="example@your.domain"
+          placeholderTextColor="#888"
+          autoComplete="email"
+          inputMode="email"
+          keyboardType="email-address"
```
Comment (marcjadaa) [inline app/screens/auth/forgot_password_screen.tsx:38]: It's cleaner than the previous version, it goes for what  is written down below too

File: app/screens/auth/set_username_screen.tsx [hunk 2]
```diff
@@ -1,135 +1,134 @@
...
+  return (
+    <View style={styles.backround}>
+      {/* Image in the backround out of the scroll view for immonility */}
+      <Image
+        source={require("@/assets/images/sign-up-screen/Ellipse 3.png")}
+        style={styles.ellipse}
+      />
+
+      <View style={styles.inputColumn}>
+        {/* Title of the screen */}
```
Comment (marcjadaa) [inline app/screens/auth/set_username_screen.tsx:28]: Good format of writing, but no changes made in the code

File: app/screens/auth/set_username_screen.tsx [hunk 3]
```diff
@@ -1,135 +1,134 @@
...
-import { View, Text, TouchableOpacity, Dimensions, StyleSheet, TextInput, Image} from 'react-native';
-import { Ionicons } from '@expo/vector-icons';
-import { useRouter } from 'expo-router';
-
-const {width, height} = Dimensions.get('window');
+import React from "react";
+import {
+  View,
+  Text,
+  TouchableOpacity,
```
Comment (marcjadaa) [inline app/screens/auth/set_username_screen.tsx:5]: This way is more visible, and help us read and understand the code better.

**Review by marcjadaa — COMMENTED** [review]
It is a better structure, visibility format of the code than the previous version committed, in addition the Node.js CI and SonarCloud checks passed, which is great!

**Review by Roman-batut — APPROVED** [review]
LGTM

**Pull request merged**